src/main/java/com/expediagroup/hiveberg/IcebergInputFormat.java

```diff
@@ -212,6 +207,14 @@ public void close() throws IOException {
     public float getProgress() throws IOException {
       return 0;
     }
+
+    private Record resolveRecord() {
```

Comment:
Spent too long thinking about a better method name for this - the purpose is to decide if we append the snapshot ID to the row or leave it as is (i.e. if we're reading from the `SnapshotTable` instead or the data table)

Reply:
Yeah it will be because we can only query from one snapshot at a time